packages/gatsby/src/utils/webpack.config.js

```diff
@@ -107,11 +107,7 @@ module.exports = async (
           // Add /* filename */ comments to generated require()s in the output.
           pathinfo: true,
           // Point sourcemap entries to original disk location (format as URL on Windows)
-          publicPath:
-            process.env.GATSBY_WEBPACK_PUBLICPATH ||
-            `${program.ssl ? `https` : `http`}://${
-              program.host
-            }:${webpackPort}/`,
+          publicPath: process.env.GATSBY_WEBPACK_PUBLICPATH || `/`,
```

Comment:
Wouldn't we still want to handle the ssl stuff? Seems like this would be a breaking change for that behavior.

If nothing else--would you be able to validate this hasn't broken that behavior?

Reply:
Just tested, works fine!

> Keep in mind that the automatic certificates issued with the --https flag are explicitly issued to localhost and will only be accepted there. Using it together with the --host option will likely result in browser warnings.

This behaviour documented in https://www.gatsbyjs.org/docs/local-https/ remains the same